internal/template.go

```diff
@@ -74,11 +77,7 @@ func execTestingTpl(data *TplData) ([]byte, error) {
 	return src, nil
 }
 
-func maybeExecBackendsTpl(tplData *TplData, bcks Backends, genBcks bool) ([]byte, error) {
-	if !genBcks {
-		return nil, nil
-	}
-
+func filterTransitiveDeps(tplData *TplData, bcks Backends) []TplDep {
```

Comment:
filter doesn't really explain whether it only returns transitive deps or filters _out_ transitive deps, i believe it's the second one, but could call it getBackendDeps and add a comment to explain?